scripts/ansible/roles/golang_base/tasks/debian-10.yml

```diff
@@ -0,0 +1,15 @@
+- name: Add Go repository
+  become: true
+  apt_repository:
+    repo: deb http://deb.debian.org/debian buster-backports main
+  tags:
+    - golang_base
+
+- name: Install Go
+  become: true
+  apt:
+    name:
+      - golang-go
```

Comment:
Same here? Will this install the desired version?

Reply:
On Debian 10 the role uses buster-backports, which is steady at 1.14.x, specifically [golang-1.14~1~bpo10+1](https://packages.debian.org/buster-backports/golang)